packages/rpc-methods/src/restricted/invokeSnap.ts

```diff
@@ -4,17 +4,20 @@ import {
   ValidPermissionSpecification,
   PermissionType,
 } from '@metamask/controllers';
-import { SnapController } from '@metamask/snap-controllers';
 import { isObject, Json, NonEmptyArray } from '@metamask/utils';
 import { ethErrors } from 'eth-rpc-errors';
-import { SNAP_PREFIX } from '@metamask/snap-utils';
+import { Snap, SNAP_PREFIX, SnapId } from '@metamask/snap-utils';
 
 const methodPrefix = SNAP_PREFIX;
 const targetKey = `${methodPrefix}*` as const;
 
 export type InvokeSnapMethodHooks = {
-  getSnap: SnapController['get'];
-  handleSnapRpcRequest: SnapController['handleRpcRequest'];
+  getSnap: (snapId: SnapId) => Snap | undefined;
+  handleSnapRpcRequest: (
```

Comment:
I'm worried we'll forget to update these 🤔

Reply:
Then it will result in an error wherever the method hooks are passed to it. If you have a better suggestion for how to fix it I'm open for that too, though!

Reply:
I'm not sure I do 😅